scripts/error-codes/codes.json

```diff
@@ -98,8 +98,8 @@
   "96": "EventPluginRegistry: Cannot inject event plugins that do not exist in the plugin ordering, `%s`.",
   "97": "EventPluginRegistry: Event plugins must implement an `extractEvents` method, but `%s` does not.",
   "98": "EventPluginRegistry: Failed to publish event `%s` for plugin `%s`.",
-  "99": "EventPluginHub: More than one plugin attempted to publish the same event name, `%s`.",
-  "100": "EventPluginHub: More than one plugin attempted to publish the same registration name, `%s`.",
+  "99": "EventPluginRegistry: More than one plugin attempted to publish the same event name, `%s`.",
+  "100": "EventPluginRegistry: More than one plugin attempted to publish the same registration name, `%s`.",
```

Comment:
I wasn't sure if we could change existing error code messages or if we had to make new ones?

Reply:
Either should be fine I think since their meaning is the same.